src/Microsoft.AspNet.Authentication.OpenIdConnect/project.json

```diff
@@ -3,8 +3,11 @@
     "description": "ASP.NET 5 middleware that enables an application to support OpenIdConnect authentication workflow.",
     "dependencies": {
         "Microsoft.AspNet.Authentication": "1.0.0-*",
+        "Microsoft.IdentityModel.Protocol.Extensions": "2.0.0-beta4-*",
+        "System.Threading.Tasks": "4.0.10-beta-*",
+        "System.Runtime": "4.0.20-beta-*",
         "Microsoft.Framework.NotNullAttribute.Sources": { "type": "build", "version": "1.0.0-*" },
-        "Microsoft.IdentityModel.Protocol.Extensions": "2.0.0-beta4-*"
+        "Microsoft.AspNet.Authentication.OAuth": "1.0.0-*"
```

Comment:
@PinpointTownes @tushargupta51 I think it helps in crafting the message url and avoiding missssspellings, since the properties like 'ClientId' avoid that category of error.